src/components/dialog/dialog-title/readme.md

```diff
@@ -0,0 +1,64 @@
+The `sbb-dialog-title` component extends the [sbb-title](/docs/components-sbb-title--docs) component. Use it in combination with the [sbb-dialog](/docs/components-sbb-dialog--docs) to display a header in the dialog with a title, a close button and an optional back button.
+
+```html
+<sbb-dialog>
+  <sbb-dialog-title accessibility-close-label="Close button" accessibility-back-label="Back button">
```

Comment:
in the case you provide an accessibility-back-label string you should also display backButton ?

Reply:
Not sure, maybe a question for @kyubisation?

Reply:
Good question. Might be an implicit opt-in.
Would make sense to me, but we should mention this in the `backButton` documentation.